chore: add local usage instructions (#69)
NASA-ACROSS/across-client: README.md

)
```

### Running the Client in Development

> If you are running a development instance of the ACROSS software system, it is straightforward to point your client installation to your local ACROSS `core-server` instance.  First, follow the server installation instructions [here](https://github.com/NASA-ACROSS/across-server#getting-started). Next, you can repoint your client from the production server to your local instance by creating a `.env` file in the root of this repository with a `HOST` variable set to your local server's host and port, typically `http://localhost:8000/api/v1`. Instantiate a new `Client` object and it should now be set up to interface with your local development server!

Reviewer: ```suggestion
If you are running a development instance of the ACROSS software system, it is straightforward to point your client installation to your local ACROSS `core-server` instance.  

  1. Follow the server installation instructions [here](https://github.com/NASA-ACROSS/across-server#getting-started). 
  2.  Copy or duplicate the `.env.example` file in the root directory and rename it to `.env`.
  3. **Important:** Override the client's host from the default production server to your local instance by setting a `HOST` variable set to your local server's host and port, default is `http://localhost:8000/api/v1`.
  4. Instantiate a new `Client` object, see the example below, and it should now be set up to interface with your local development server!
```

desired: I find these kinds of instructions easier to follow as numbered steps. I also adjusted some of the wording to be a little more direct.

Using this local setup, you can also test submitting schedules to ACROSS. The local server provides a seeded service account with schedule write permissions. Simply instantiate the client using these credentials and you can post a schedule to your local server:
```py
from across.client import Client

ACROSS_SERVER_ID = "961fbb3b-dd5e-45ff-a282-41033e43933d"
ACROSS_SERVER_SECRET = "prehibernationweek"

client = Client(
   client_id=ACROSS_SERVER_ID,
   client_secret=ACROSS_SERVER_SECRET,
)

> client.schedule.post(...)

Reviewer: ```suggestion
created_schedule_id = client.schedule.post(...)

schedule = client.schedule.get(created_schedule_id)

print(schedule)
```

desired: i'm not 100% if this is how the get works, but, i think showing that the schedule was successfully created and you can use the client to get it, shows the complete flow?

```

> For ease of use, `ACROSS_SERVER_ID`  and `ACROSS_SERVER_SECRET` can be added to the root `.env` file as well.

Reviewer: ```suggestion
For ease of use, `ACROSS_SERVER_ID`  and `ACROSS_SERVER_SECRET` are part of the `.env.example` file as well.
```
